docs/docs/sample-scripts/flow/export-flow-logicapp/index.mdx

```diff
@@ -37,7 +37,7 @@ This script will export the Power Automate flow *Your sample test flow*, make su
 
   ```bash
   #!/bin/bash
-  ENV_NAME=m365 flow environment list --query '[?contains(displayName,`default`)] .name'
+  ENV_NAME=m365 flow environment get --query 'name'
```

Comment:
I would also add `output` mode to `json` as script sample should not assume we use some output mode. For example someone might have `text` set as default and then this would probably break

```suggestion
  ENV_NAME=m365 flow environment get -o json --query 'name'
```